src/Uno.UWP/UI/Input/GestureRecognizer.Manipulation.cs

```diff
@@ -274,6 +313,49 @@ private ManipulationDelta GetDelta(ManipulationDelta cumulative)
 				};
 			}
 
+			// For pen and mouse this only means down -> * moves out of tap range;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270104)